bundle/manifests/opendatahub-operator.clusterserviceversion.yaml

```diff
@@ -814,6 +814,7 @@ spec:
           - gatewayclasses
           - gateways
           - httproutes
+          - referencegrants
```

Comment:
not sure i understand.
which resource we are doing cross reference? and where we created ReferenceGrant in the PR?
isnt that to pass a different value to dashboard-url is all for this PR?

Reply:
We need to create an HTTPRoute for the dashboard service so that the client can access the dashboard via the gateway. Since dashboard service in opendatahub namespace, i am creating a HTTPRoute in opendatahub namespace. But the gateway is in openshigt-ingress namespace, for the cross namespace compatibility we need to create referencegrants.

Reply:
https://github.com/opendatahub-io/opendatahub-operator/pull/2569/files#diff-a890b97803886fe0ed3dd18f04b31d5eaf89c7398f3b02a7026f478ab4b8c620R108

Reply:
I agree with @zdtsw here. If the plan is to add it in another PR, hence, the permissions should be add there as well.